tsdb/index/tsi1/partition.go

```diff
@@ -480,6 +491,11 @@ func (p *Partition) prependActiveLogFile() error {
 		return err
 	}
 	p.manifestSize = manifestSize
+
+	// Set the file metrics again.
```

Comment:
Maybe a bug... this block is almost an exact match to the one above, but it's missing:
```go
p.tracker.SetSeries(p.seriesIDSet.Cardinality())
```